interpreter.ts

```diff
@@ -106,8 +106,9 @@ module J2ME {
   var argArray = [];
 
   function buildExceptionLog(ex, stackTrace) {
-    var className = ex.klass.classInfo.className;
-    var detailMessage = util.fromJavaString(ex.klass.classInfo.getFieldByName("detailMessage", "Ljava/lang/String;", false).get(ex));
+    var className = ex.klass.classInfo.getClassNameSlow();
+    var classInfo: ClassInfo = ex.klass.classInfo;
+    var detailMessage = util.fromJavaString(classInfo.getFieldByName(toUTF8("detailMessage"), toUTF8("Ljava/lang/String;"), false).get(ex));
```

Comment:
Nit: this function defines a *classInfo* local and then uses it only once, even though there are two references to *ex.klass.classInfo* in this function (the second one being the *getClassNameSlow* call above this line). The function should use the *classInfo* local for both references.